lib/model/entity/fiscaal/SaldoCommissie.enum.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * SaldoCommissie.enum.php
+ *
+ * @author G.J.W. Oolbekkink <g.j.w.oolbekkink@gmail.com>
+ *
+ */
+abstract class SaldoCommissie implements PersistentEnum {
```

Comment:
Dit is nu zo gedaan omdat er onderscheid gemaakt wordt tussen twee totaal verschillende manieren van saldo verwerken. We zijn er nu mee bezig om deze twee systemen meer op elkaar te laten lijken.